prow/plugins/trickortreat/.gitignore

```diff
@@ -0,0 +1 @@
+api-key
```

Comment:
Why do we need this if we aren't calling an API?

Reply:
great catch, deleted